src/eu/eutampieri/catacombs/model/Enemy.java

```diff
@@ -1,10 +0,0 @@
-package eu.eutampieri.catacombs.model;
-
-public class Enemy extends GameObject{ //general enemy for example purpose
-
-	public Enemy(int x, int y, ID id) {
-		super(x, y, id);
-		
-	}
-
-}
```

Comment:
Immagino che adesso per distinguere se sia un nemico o meno si debba guardare l'ID, che a questo punto non chiamerei ID

Reply:
nono i nemici hanno un nome ora tipo slime e bat

Reply:
Si, intendo che se vuoi vedere se è un nemico devi controllare il campo `id` di tipo `ID`. Giusto?
Niente in contrario a quello che hai fatto, anzi mi piace.
L'unica cosa è che id non lo chiamerei id ma qualcosa che ricordi il "tipo" di oggetto, tipo nemico, player,...

Reply:
tipo chiamarlo entityType?